src/beautifiers/ocamlformat.coffee

```diff
@@ -28,10 +28,9 @@ module.exports = class OCamlFormat extends Beautifier
     OCaml: true
   }
 
-  beautify: (text, language, options) ->
-    @run("ocamlformat", [
-      @tempFile("input", text)
-      ], {
+  beautify: (text, language, options, context) ->
+    args = if context.filePath then ["--name", context.filePath] else []
+    @run("ocamlformat", args.concat([@tempFile("input", text)]), {
```

Comment:
Should an extension be added to the `tempFile` generated? See https://github.com/Glavin001/atom-beautify/blob/bb9428a4fa483fa6ffe958cdcb519b27e00cb770/src/beautifiers/phpcbf.coffee#L61 for example:

```suggestion
    @run("ocamlformat", [@tempFile("input", text, ".ml")], {
```

Or needs to use the extension of the `context.filePath`? See:
- https://github.com/Glavin001/atom-beautify/blob/bb9428a4fa483fa6ffe958cdcb519b27e00cb770/src/beautifiers/uncrustify/index.coffee#L50
- https://github.com/Glavin001/atom-beautify/blob/bb9428a4fa483fa6ffe958cdcb519b27e00cb770/src/beautifiers/uncrustify/index.coffee#L104

```suggestion
    fileExtension = context.fileExtension
    @run("ocamlformat", args.concat([@tempFile("input", text, fileExtension and ".#{fileExtension}")]), {
```

Or would you recommend still using the `--name` CLI argument instead?

Reply:
It needs to use the extension of the current file. I did something similar to your second suggestion first but the authors of ocamlformat suggested `--name` since it reads and follows the correct ocamlformat config file (`.ocamlformat`) as well.